makefile.shade

```diff
@@ -151,27 +165,21 @@ var buildTarget = "compile"
                 }
             });
         }
+        
+        var commitsAsString = string.Join(Environment.NewLine, commits.Select(c => c.Key + ":" + c.Value));
+        File.WriteAllText(Path.Combine(universeArtifacts, "commits"), commitsAsString);
     }
-
-#smoke-test-mono .pull .fix-project-json .change-default-build-target-to-verify .build-all
-
-#fix-project-json
+    
+#remove-src-folders
     @{
-        repositories = repositories.Except(excludeReposOnMono).ToArray();
+        foreach (var repo in GetRepositoriesToBuild())
```

Comment:
Deleting sources wouldn't work if projects have `target: project` https://github.com/aspnet/PlatformAbstractions/blob/dev/test/QuickTest/project.json#L8. You might have to additionally address that.